install

```diff
@@ -18,7 +19,8 @@ startup_dir = os.path.join(profile_dir, 'startup')
 shutil.copy(config_file, profile_dir)
 # Then, generate the custom imports file
 sp.call([os.path.join(rundir, 'generate_failover_imports'),
-         import_file, output_file])
+        import_file, output_file])
```

Comment:
This line should align with the square, not circle, bracket (because it's still part of the list)